sentry-log4j2/src/test/kotlin/io/sentry/log4j2/SentryAppenderTest.kt

```diff
@@ -591,4 +589,27 @@ class SentryAppenderTest {
         }
       )
   }
+
+  @Test
+  fun `sets properties from ThreadContext as attributes on logs`() {
+    val logger = fixture.getSut(minimumLevel = Level.INFO, contextTags = listOf("someTag"))
+    ScopesAdapter.getInstance().options.logs.isEnabled = true
```

Comment:
`l` Is this line even needed? Afair we're setting `logs.enabled=true` in `sentry.properties` which should take effect here.